### ISSUE TITLE
evaluation scores are low

### ISSUE THREAD

**Issue by sangtrx** [ttengwang/dense-video-captioning-pytorch#3]
Hi, I have a question about the result in the paper. When I tried to replicate, my result is lower than what is shown in the paper. My assumption is the author multiplied what he got with 1000. Can anyone help me to clarify this?
Thank you in advance

my result: 
{'Bleu_1': 0.05203881419889433, 'Bleu_2': 0.02532179465691948, 'Bleu_3': 0.009912868169621145, 'Bleu_4': 0.002988311654355243, 'METEOR': 0.02982898949811799, 'ROUGE_L': 0.05281023682433468, 'CIDEr': 0.054126791927464786, 'Recall': 0.2033760423022168, 'Precision': 0.2033760423022168, 'tiou': 0.6}

**Comment by ttengwang** [comment]
Hi, multiply the above scores by 4917/1000 and you will get similar results to the paper. Since I use annotations of the full validation set (4917 videos) as references, only 1000 videos have a nonzero score and others get zeros.

**Comment by sangtrx** [comment]
Hi, thank you for your reply
When I multiply METEOR score with 4.917, I can get the same result but what about Bleu_4 and CIDEr?

Here is the scores on the pretrain model:
{'Bleu_1': 0.05159802286213217, 'Bleu_2': 0.024580457878963728, 'Bleu_3': 0.009422214031248557, 'Bleu_4': 0.002974386088004809, 'METEOR': 0.02950847582339392, 'ROUGE_L': 0.052153710311550236, 'CIDEr': 0.05126019102296149, 'Recall': 0.2033760423022168, 'Precision': 0.2033760423022168, 'tiou': 0.6}

**Comment by ttengwang** [comment]
For ground-truths proposals, METEOR 14.51, CIDEr 25.20, Bleu_4 1.46
For learnt proposals, METEOR 10.14, CIDEr 9.25, Bleu_4 0.72

Cider and Bleu are low because we only use METEOR as the reward in SCST.
